x-pack/legacy/plugins/canvas/public/plugin.tsx

```diff
@@ -79,6 +83,9 @@ export class CanvasPlugin
         return renderApp(coreStart, depsStart, params, canvasStore);
       },
     });
+
+    plugins.home.featureCatalogue.register(featureCatalogueEntry);
```

Comment:
Isn't https://github.com/elastic/kibana/pull/56755/files#diff-2001166b54e08ea07258b65005363ee9R14 already taking care of registering the entry?